server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -182,4 +182,6 @@ grant {
   permission java.io.FilePermission "/sys/fs/cgroup/memory", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/memory/-", "read";
 
+  // ZSTD permissions
+  permission java.lang.RuntimePermission "*";
```

Comment:
:( Please fix that to have narrowed set of permissions

Reply:
+1; we can't have the security here be wide open

Reply:
Yes it is a bit complex thing. What would be better:
- change it to `java.lang.RuntimePermission "loadLibrary.*"`
- Alternative solution is to re-pack zstd library and add *.so files in the `libs` folder?